src/main/java/Duke.java

```diff
@@ -7,52 +7,87 @@ public static void main(String[] args) {
                 + "|  o  /|     ||  o  /|     |\n"
                 + "|  o  \\|  o  ||  o  \\|  o  |\n"
                 + "|_____/|_____||_____/|_____|\n";
-        String HORIZONTAL_LINE = "____________________________________________________________";
+        String horizontalLine = "____________________________________________________________";
 
+        Task[] tasks = new Task[100];
+        int numberOfTasks = 0;
         String line;
         boolean isRunning = true;
         Scanner in = new Scanner(System.in);
 
         System.out.println(Logo);
-        System.out.println(HORIZONTAL_LINE);
+        System.out.println(horizontalLine);
         System.out.println("Hello! I'm Bobo!");
         System.out.println("I'm a little blur, but I'd love to help!");
-        System.out.println(HORIZONTAL_LINE);
+        System.out.println(horizontalLine);
 
         while (isRunning) {
             line = in.nextLine();
 
             if (line.equals("bye")) {
                 isRunning = false;
             } else if(line.equals("list")) {
-                System.out.println(HORIZONTAL_LINE);
-                for (int i = 0; i < Task.numberOfTasks; i++) {
-                    System.out.println((i + 1) + ".["
-                            + Task.getTask(i).getStatusIcon() + "] "
-                            + Task.getTask(i).getDescription());
+                System.out.println(horizontalLine);
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < numberOfTasks; i++) {
+                    System.out.println((i + 1) + "." + tasks[i]);
                 }
-                System.out.println(HORIZONTAL_LINE);
+                System.out.println(horizontalLine);
             } else if (line.startsWith("done")) {
                 int taskNumber = Integer.parseInt(line.substring(4).trim()) - 1;
-                Task.getTask(taskNumber).markAsDone();
+                tasks[taskNumber].markAsDone();
 
-                System.out.println(HORIZONTAL_LINE);
+                System.out.println(horizontalLine);
                 System.out.println("Okie! Marked this as done: ");
-                System.out.println((taskNumber + 1) + ".["
-                        + Task.getTask(taskNumber).getStatusIcon() + "] "
-                        + Task.getTask(taskNumber).getDescription());
-                System.out.println(HORIZONTAL_LINE);
-            } else {
-                Task t = new Task(line);
+                System.out.println((taskNumber + 1) + "." + tasks[taskNumber]);
+                System.out.println(horizontalLine);
+            } else if (line.startsWith("todo")) {
+                String description = line.substring(5);
```

Comment:
You can consider defining what is 5! For e.g. splitIndex = 5 (where the '/' is located)